### PR TITLE
Update SECURITY.md

### PR DESCRIPTION
**Pull request by RichiH**
Signed-off-by: Richard Hartmann <richih@richih.org>

### CODE REVIEW

File: SECURITY.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ Please encrypt your message to us; please use our PGP key. The key fingerprint i
 
 F988 7BEA 027A 049F AE8E  5CAA D125 8932 BE24 C5CA
 
-The key is available from [pgp.mit.edu](https://pgp.mit.edu/pks/lookup?op=get&search=0xF9887BEA027A049FAE8E5CAAD1258932BE24C5CA) by searching for [grafana](https://pgp.mit.edu/pks/lookup?search=grafana&op=index).
+The key is available from [pgp.mit.edu](http://keys.gnupg.net/pks/lookup?search=0xF9887BEA027A049FAE8E5CAAD1258932BE24C5CA&fingerprint=on&op=index) by searching for [grafana](https://pgp.mit.edu/pks/lookup?search=grafana&op=index).
```
Comment (sakjur) [inline SECURITY.md:9]:
```suggestion
The key is available from [keys.gnupg.net](http://keys.gnupg.net/pks/lookup?search=0xF9887BEA027A049FAE8E5CAAD1258932BE24C5CA&fingerprint=on&op=index) by searching for [grafana](https://pgp.mit.edu/pks/lookup?search=grafana&op=index).
```

Comment (sakjur) [inline SECURITY.md:9]: Getting the keys via HTTP seems very bad and their HTTPS is broken for me 😢

Comment (RichiH) [reply SECURITY.md:9]: Fair.

**Comment by RichiH** [comment]
Did https://github.com/grafana/grafana/pull/25664 from CLI.

**Pull request closed without merging**